changelog.d/2-features/pr-3202

```diff
@@ -0,0 +1 @@
+`conversations/join` endpoint rate limited per IP address
```

Comment:
I suppose this introduces the conversations/join endpoint in the public API. The rate limit is secondary. Can you reformulate and move this to API changes subfolder?